autofaiss/utils/decorators.py

```diff
@@ -21,15 +24,18 @@ def __enter__(self):
             if self.comment is not None:
                 space = "\t" * self.indent
                 start_date = datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
-                print(f"{space}{self.comment} {start_date}", flush=True)
+                logger.info(f"{space}{self.comment} {start_date}")
+                # previously used print had flush=True, this flushes all handlers
```

Comment:
This line is a note explaining the commit rather than a note describing the code, maybe we should delete it?
We flush here to make sure that we display the log in stdout before entering in the wrapped function